engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java

```diff
@@ -21,6 +21,7 @@ private SortableFieldMetadata(String fieldName) {
 
 	/**
 	 * The name of the field whose field bridge will be used to determine the value for the doc value field.
+	 * @return the name of the field
```

Comment:
Right, I made confusion again.
The problem is that, if there is a different error in the build, this appear in the log in the following way:
```
[ERROR] Exit code: 1 - /home/ddalto/workspace/projects/hibernate-search/engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java:25: warning: no @return
```

If other people reading thi s don't spot it, the warning symbol is at the end of the sentence.

I dislike these kind of comments as well but I don't want to be in the situation where I have a huge list of warning and then I need to look for the real error.
I also tend to forget to pay close attention to the bug 
I would prefer if we can agree on a way to remove the warning as well for these cases.

Maybe the following is better?
```
	/**
	 * @return the name of the field whose field bridge will be used to determine the value for the doc value field.
	 */
	public String getFieldName() {
```